modules/tensor_mechanics/doc/content/source/kernels/MechanicalBodyForce.md

```diff
@@ -0,0 +1,17 @@
+# MechanicalBodyForce
+
+!syntax description /Kernels/MechanicalBodyForce
+
+## Description
+
+The kernel `MechanicalBodyForce` provides a body force term in the stress divergence equilibrium.
```

Comment:
We should say something here about the fact that it's prescribed using a material property.

Reply:
This should also link to the page for the Action as a suggestion for how to use this more easily.